py/selenium/webdriver/safari/webdriver.py

```diff
@@ -107,23 +66,24 @@ def quit(self):
         except http_client.BadStatusLine:
             pass
         finally:
-            if not self._reuse_service:
```

Comment:
figure out what to do with `reuse_service` and factor it in; should it be responsibility of `Service`? (weird api?)